service/src/test/java/uk/gov/hmcts/reform/fpl/utils/NotifyAdminOrderIssuedTestHelper.java

```diff
@@ -0,0 +1,119 @@
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.ISO_8859_1;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.IssuedOrderType.CMO;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
+
+public class NotifyAdminOrderIssuedTestHelper {
+
+    private NotifyAdminOrderIssuedTestHelper() {
+    }
+
+    private static final byte[] PDF = {1, 2, 3, 4, 5};
+    private static final String LOCAL_AUTHORITY_NAME = "Example Local Authority";
+
+    public static MapDifference<String, Object> verifyNotificationSentToAdminWhenOrderIssued(
+        ArgumentCaptor<Map<String, Object>> dataCaptor, IssuedOrderType issuedOrderType) {
+
+        Map<String, Object> resultData = new HashMap<>(dataCaptor.getValue());
+        Map<String, Object> expectedParameters = issuedOrderType == CMO
+            ? getExpectedCMOParametersForAdminWhenRepresentativesNeedServingByPost() :
+            getExpectedParametersForAdminWhenRepresentativesNeedServingByPost();
+
+        assertThat(((JSONObject) resultData.get("caseUrlOrDocumentLink")).get("file")).isEqualTo(
+            ((JSONObject) expectedParameters.get("caseUrlOrDocumentLink")).get("file"));
+
+        resultData.remove("caseUrlOrDocumentLink");
+        expectedParameters.remove("caseUrlOrDocumentLink");
+
+        return Maps.difference(expectedParameters, resultData);
+    }
+
+    public static Map<String, Object> getExpectedParametersForAdminWhenNoRepresentativesServedByPost() {
+        return ImmutableMap.<String, Object>builder()
+            .put("cmoCallout", "")
+            .putAll(commonParametersNoPostingNeeded())
+            .build();
+    }
+
+    public static Map<String, Object> getExpectedCMOParametersForAdminWhenNoRepresentativesServedByPost() {
+        return ImmutableMap.<String, Object>builder()
+            .put("cmoCallout", "^Jones, SACCCCCCCC5676576567")
+            .putAll(commonParametersNoPostingNeeded())
+            .build();
+    }
+
+    public static List<Element<Representative>> buildRepresentativesServedByPost() {
+        return wrapElements(Representative.builder()
+            .email("paul@example.com")
+            .fullName("Paul Blart")
+            .address(Address.builder()
+                .addressLine1("Street")
+                .postTown("Town")
+                .postcode("Postcode")
+                .build())
+            .servingPreferences(POST)
+            .build());
+    }
+
+    private static Map<String, Object> commonParametersNoPostingNeeded() {
+        return ImmutableMap.<String, Object>builder()
```

Comment:
Map.of()

Reply:
No nicer way of doing this as far as I'm aware, @lewisbirks agrees

Reply:
Here `Map.of()` could be used, in the other [situation](https://github.com/hmcts/fpl-ccd-configuration/pull/870/files/c17124d9faaa9df50e8820757420c7d7fbe14f3e#r379378335) where you are adding the contents of another map then `Map.of()` cannot be used

Reply:
yep I just realised too, changed 👍